src/main/java/bio/terra/service/dataset/flight/ingest/IngestPopulateFileStateFromFlightMapAzureStep.java

```diff
@@ -22,8 +23,9 @@ public IngestPopulateFileStateFromFlightMapAzureStep(
       ObjectMapper objectMapper,
       Dataset dataset,
       int batchSize,
-      Predicate<FlightContext> skipCondition) {
-    super(loadService, objectMapper, dataset, batchSize, skipCondition);
+      Predicate<FlightContext> skipCondition,
+      FileService fileService) {
```

Comment:
It looks like we usually group injected objects like `FileService` at the front of the parameter list.

Reply:
Ack. IntelliJ's auto-refactor doesn't know to order things like we want to order them